plugin/pxr/cmake/macros/Public.cmake

```diff
@@ -689,7 +689,7 @@ function(pxr_toplevel_prologue)
             set_target_properties(usd_ms
                 PROPERTIES
                     FOLDER "${folder}"
-                    PREFIX "${PXR_LIB_PREFIX}"
+                    PREFIX "${CMAKE_SHARED_LIBRARY_PREFIX}"
```

Comment:
This change makes sense, though since CMAKE_SHARED_LIBRARY_PREFIX is the default value for PREFIX, we could drop the setting altogether.